articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+  domain:       'login.northwind.com',
+  clientID:     'your-client-id'
+});
+```
+
+Note that the Management API only accepts Auth0 domains. If you use a [custom domain](/custom-domains) and also intend to perform [Management API actions with Auth0.js](/libraries/auth0js/v9#user-management), you must instantiate a new copy of `webAuth` using your Auth0 domain.
+
+## Auth0 emails
+
+If you would like your custom domain used with your Auth0 emails, you'll need to enable this feature in the [Dashboard](${manage_url}/#/tenant). You can do this by clicking the toggle associated with the **Use Custom Domain in Emails**. When the toggle is green, this feature is enabled.
+
+![](/media/articles/custom-domains/cd_email_toggle.png)
+
+## Social identity provider configuration
+
+If you want to use social identity providers with your custom domain, you must update the allowed callback URLs to include your custom domain (such as `https://login.northwind.com/login/callback`).
+
+::: warning
+You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
+:::
+
+## SAML identity provider configuration
+
+If you want to use SAML identity providers with your custom domain, you must obtain the service provider metadata from Auth0 (such as `https://login.northwind.com/samlp/metadata?connection=CONNECTION_NAME`). This includes updated Assertion Consumer Service (ACS) URLs. You will then need to manually update this value in your IdP(s). This change to your IdP(s) will need to happen at the same time as you begin using your custom domain in your applications. This can pose a problem if there are multiple IdPs to configure.
+
+Alternatively, you can use signed requests to fulfill this requirement. Once your custom domain is set up, in the [Dashboard](${manage_url}/#/tenant/custom_domains), you will need to download the certificate from the link under the “Sign Request” toggle. You can then give the certificate to the IdP(s) to upload. This enables the IdP to validate the signature on the AuthnRequest message that Auth0 sends to the IdP. The certificate needs to be imported into the IdP and, if necessary, signature verification should be enabled (exact steps vary by IdP).
+
+You should then turn on the “Sign Request” toggle in the Dashboard under **Connections > Enterprise > SAMLP > CONNECTION**. This will trigger Auth0 to sign the SAML AuthnRequest messages it sends to the IdP.
+
+Once this is done, and you use your custom domain when interacting with Auth0, the IdP will receive that custom domain in your signed request. Because your application is trusted, the IdP should automatically override whatever was configured as your ACS URL and replace it with the value sent in the signed request. This will prevent you from having to change one or many IdP settings all at the same time, instead allowing you to prepare them to accept your signed requests ahead of time. you can then at a later date have the IdPs change the ACS URL as well.
+
```

Comment:
@jeffreylees If you would please add a note that indicates the Try button will not work in Dashboard if your IDP is configured to use the custom domain. Thanks!